engine/security-api/src/main/scala/pl/touk/nussknacker/ui/security/api/LoggedUser.scala

```diff
@@ -2,14 +2,37 @@ package pl.touk.nussknacker.ui.security.api
 
 import pl.touk.nussknacker.ui.security.api.Permission.Permission
 
-case class LoggedUser(
-  id: String,
-  categoryPermissions: Map[String, Set[Permission]] = Map.empty,
-  isAdmin: Boolean = false
-) {
-  private val permissions = categoryPermissions.values.flatten.toSet
-
-  def hasPermission(permission: Permission): Boolean = {
-    isAdmin || permissions.contains(permission)
+sealed trait LoggedUser {
+  val id: String
+  def hasPermission(category: String, permission: Permission): Boolean
```

Comment:
can we "hide" it somehow, so the only api for checking permission is u.can(c, p) ?

Reply:
I was actually thinking about removing `PermissionSyntax` and renaming `hasPermission` -> `can`. I'll do it that way.

Reply:
either way is good

Reply:
done